app/authentication/auth.py

```diff
@@ -92,33 +92,42 @@ def requires_auth():
             err_msg = "Invalid token: algorithm used is not HS256"
             raise AuthError(err_msg, 403, service_id=service.id, api_key_id=api_key.id)
         except TokenDecodeError:
+            # we attempted to validate the token but it failed meaning it was not signed using this api key.
+            # Let's try the next one
+            # TODO: Change this so it doesn't also catch `TokenIssuerError` or `TokenIssuedAtError` exceptions (which
+            # are children of `TokenDecodeError`) as these should cause an auth error immediately rather than
+            # continue on to check the next API key
             continue
+        except TokenError:
+            # General error when trying to decode and validate the token
+            err_msg = 'Invalid token: API token is not valid'
```

Comment:
@karlchillmaid in this case, the error is supposed to act as the catch all where we can't give more details. Therefore the best we can do now is to hand them off to the guidance e.g.

`Invalid token. Make sure your API token matches our guidance at https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header`

Reply:
How about `Invalid token: Make sure your API token matches the example at https://...`?